audbackend/core/conftest.py

```diff
@@ -1,29 +1,36 @@
+import os
 import tempfile
+
 import pytest
 
 import audbackend
 import audeer
 
 
-@pytest.fixture(autouse=True)
-def create_backend(doctest_namespace):
+@pytest.fixture(scope='function', autouse=True)
```

Comment:
Is scope `function` correct, or can this be reused across functions?

Reply:
I think it makes writing the tests easier if we always start from a fresh repository.

Reply:
If it does not slow down the tests considerable, it seems indeed the safer option.
Have you any experience how long it takes to create the repo?

Reply:
~1-2 seconds

Reply:
OK, not ideal but as it is used for the docstring tests where we shouldn't expect a big increase in tests it should be fine.

Reply:
Oh wait a moment. You were actually referring to the docstring examples here, which is using a file-system backend only. This has almost no overhead at all.